manageiq-operator/api/v1alpha1/helpers/miq-components/httpd.go

```diff
@@ -73,6 +73,8 @@ func Route(cr *miqv1alpha1.ManageIQ, scheme *runtime.Scheme, client client.Clien
 		}
 
 		route.Spec.Host = cr.Spec.ApplicationDomain
+		route.Spec.TLS.Certificate = ""
+		route.Spec.TLS.Key = ""
```

Comment:
I don't know if people would want to manually set certificates on routes, but this would remove them and force the use of the cluster certificates.

Reply:
Can you comment as such right on the code?

Reply:
What happens if customer provides a custom tls cert secret in the IMInstall CR? Will custom cert still work?

Reply:
Done

Reply:
> What happens if customer provides a custom tls cert secret in the IMInstall CR? Will custom cert still work?

No, this would force using the certificates on the cluster.
Wouldn't they install that certificate on the cluster?

Reply:
Not sure how this works. Probably. I am ok with the fix